server/service/integration_live_queries_test.go

```diff
@@ -157,7 +157,9 @@ func (s *liveQueriesTestSuite) TestLiveQueriesRestOneHostOneQuery() {
 		var cid string
 		cidChannel := make(chan string)
 		go func() {
-			for {
+			ticker := time.NewTicker(100 * time.Millisecond)
+			defer ticker.Stop()
+			for range ticker.C {
```

Comment:
Do we go into the for loop right away or wait 100 ms before the 1st entry?

Reply:
Waits 100ms